### DISCUSSION TITLE
New Device File Support?

### DISCUSSION THREAD

**Discussion by Hajman** [zwave-js/zwave-js#2086] [General]
First, thanks for this excellent project to date.  It is a step up from Official Zwave and OZW add-ons.

Any update on the implementation of https://github.com/zwave-js/node-zwave-js/issues/1600

**Comment by AlCalzone** [comment]
> Any update on the implementation of #1600

Slowly but steady. We're dealing with a lot of unreviewed files and cleaning them up is a lot of manual work. If you want to help out, you can join our [Slack](https://join.slack.com/t/zwave2mqtt/shared_invite/enQtNjc4NjgyNjc3NDI2LTc3OGQzYmJlZDIzZTJhMzUzZWQ3M2Q3NThmMjY5MGY1MTc4NjFiOWZhZWE5YjNmNGE0OWRjZjJiMjliZGQyYmU) and ping @blhoward2
